js/historyStates.js

```diff
@@ -1,84 +1,73 @@
-function History_States(maxSize) {
-    let arr = new Array(CELLS_PER_ROW * CELLS_PER_ROW).fill(CANVAS_INIT_COLOR);
-    this.array = [arr];        // empty array
-    this.ptr = 0;              // pointer
-    this.maxSize = maxSize;    // maximum size
-}
-History_States.prototype.getCurrentState = function()
-{
-    return this.array[this.ptr];
-}
-
-History_States.prototype.decPtr = function()
-{
-    if(this.ptr <= 0)
-        return;
-
-    this.ptr--;
-};
-History_States.prototype.incPtr = function()
-{
-    if(this.ptr >= this.array.length-1)
-        return;
+class History_States {
```

Comment:
@Viet281101  What was the reason for converting this into a class? What are the benefits?

I am open to upgrading the style of this project as well I know what the `Pascal_Case_Is_A_Little_Unconventional` 😆

Reply:
@Viet281101 This conversation was marked as resolved but I didn't see a response to this. Can you let me know what you think?

Posting the image in case you don't see what I am seeing:

<img width="786" alt="image" src="https://github.com/Kully/pixel-paint/assets/10369095/b6296043-36e1-4377-8048-f30b154b5ce2">